src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/Behandling.kt

```diff
@@ -0,0 +1,223 @@
+package no.nav.familie.ks.sak.kjerne.behandling
+
+import no.nav.familie.ks.sak.common.Feil
+import no.nav.familie.ks.sak.common.entitet.BaseEntitet
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingType.FØRSTEGANGSBEHANDLING
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingType.REVURDERING
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingType.TEKNISK_ENDRING
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.ENDRET_UTBETALING
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.FORTSATT_INNVILGET
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.FORTSATT_OPPHØRT
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.HENLAGT_FEILAKTIG_OPPRETTET
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.HENLAGT_SØKNAD_TRUKKET
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.HENLAGT_TEKNISK_VEDLIKEHOLD
+import no.nav.familie.ks.sak.kjerne.behandling.Behandlingsresultat.IKKE_VURDERT
+import no.nav.familie.ks.sak.kjerne.fagsak.Fagsak
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.SequenceGenerator
+import javax.persistence.Table
+import no.nav.familie.kontrakter.felles.oppgave.Behandlingstype as OppgaveBehandlingType
+
+@Entity(name = "Behandling")
+@Table(name = "BEHANDLING")
+data class Behandling(
```

Comment:
Hva er egentlig grunnen til at man har brukt dataklasser som entitetklasse i familie-ba-sak?
Trodde egentlig at det var en ikke-ting å gjøre: https://stackoverflow.com/questions/58127353/should-i-use-kotlin-data-class-as-jpa-entity

Har jo fungert fram til nå, men bare et spørsmål egentlig 😛